docbook/Admin_Guide/en-US/config/display.xml

```diff
@@ -194,5 +194,14 @@
 				</para>
 			</listitem>
 		</varlistentry>
+		<varlistentry>
+			<term>$g_markdown_enabled</term>
```

Comment:
If we end up removing this config option, then we should revert the change to this file.